### ISSUE TITLE
Wrong URL in update information?

### ISSUE THREAD

**Issue by probonopd** [Nitrux/znx#45]
```
me@host:~$ dd if=/isodevice/boot/iso/nitrux_release_testing.iso  bs=1 skip=33651 count=512 2>/dev/null
http://88.198.66.58:8000/nitrux_release_testing.iso.zsync
```

but

http://88.198.66.58:8000/nitrux_release_testing.iso.zsync

does not seem to exist... (my `nitrux_release_testing.iso` is from October 2018)

**Comment by lslvr** [comment]
Indeed, it no longer exists on our server. That's because we remove ISOs not associated to a `git` branch. 😅

**Comment by probonopd** [comment]
What does `me@host:~$ dd if=/isodevice/boot/iso/nitrux_release_testing.iso  bs=1 skip=33651 count=512 2>/dev/null` give on the latest Nitrux version? (Don't have one at hand)

**Comment by lslvr** [comment]
We don't have a testing branch anymore. These are the existing branches: https://github.com/Nitrux/nitrux-iso-tool/branches. So you'd need to use another ISO.

P.S: This issue doesn't belong here.
